cmd/export_accounts.go

```diff
@@ -24,79 +22,56 @@ should be used in an initial data dump. In order to get account information with
 the export_ledger_entry_changes command.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		cmdLogger.SetLevel(logrus.InfoLevel)
-		endNum, useStdout, strictExport, isTest := utils.MustCommonFlags(cmd.Flags(), cmdLogger)
+		endNum, strictExport, isTest := utils.MustCommonFlags(cmd.Flags(), cmdLogger)
+		cmdLogger.StrictExport = strictExport
```

Comment:
Do we ever do anything with the `StrictExport` flag? Is there merit in us even keeping this bool in the code? Afaik I can't see the utility for it.

Reply:
I think what we get out of it is to choose what the tool does when an entry during export fails. The tool crashes with 'strict_export'. Without it, it logs an error and continues onto the next item instead. We should probably decide whether there's a reason to make this behavior customizable, or if only one of the behaviors are needed.

I think we want `strict_export` behavior (failing loudly) in prod. I can imagine allowing failures to pass through maybe being useful in dev, but unsure if that will really be the case. What do you think?

Reply:
I agree. That is useful and should be set in the production env